[debug] [2018-02-23T21:34:24.626Z] ----------------------------------------------------------------------
[debug] [2018-02-23T21:34:24.628Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve
[debug] [2018-02-23T21:34:24.628Z] CLI Version:   3.17.4
[debug] [2018-02-23T21:34:24.628Z] Platform:      darwin
[debug] [2018-02-23T21:34:24.628Z] Node Version:  v9.5.0
[debug] [2018-02-23T21:34:24.628Z] Time:          Fri Feb 23 2018 15:34:24 GMT-0600 (CST)
[debug] [2018-02-23T21:34:24.628Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-02-23T21:34:24.665Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-02-23T21:34:24.665Z] > authorizing via signed-in user
[debug] [2018-02-23T21:34:24.666Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2018-02-23T21:34:24.667Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 { refresh_token: '1/FsaD7rQd5wzsAHHUmlxLU1C1zTq3bQjTC7hR7UtMxWg',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  grant_type: 'refresh_token',
  scope: 'email https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/cloudplatformprojects.readonly https://www.googleapis.com/auth/firebase openid' } 
 Fri Feb 23 2018 15:34:24 GMT-0600 (CST)
[debug] [2018-02-23T21:34:26.556Z] <<< HTTP RESPONSE 200
[debug] [2018-02-23T21:34:26.625Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/conectodos-5d860  
  
 Fri Feb 23 2018 15:34:26 GMT-0600 (CST)
[debug] [2018-02-23T21:34:28.429Z] <<< HTTP RESPONSE 200
[debug] [2018-02-23T21:34:28.429Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/database/conectodos-5d860/tokens  
  
 Fri Feb 23 2018 15:34:28 GMT-0600 (CST)
[debug] [2018-02-23T21:34:29.542Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/Users/arandano004/Google Drive/IoTTech/Github/firebase/conectodos'...
[info] 
[debug] [2018-02-23T21:34:29.544Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/conectodos-5d860  
  
 Fri Feb 23 2018 15:34:29 GMT-0600 (CST)
[debug] [2018-02-23T21:34:30.997Z] <<< HTTP RESPONSE 200
[debug] [2018-02-23T21:34:30.998Z] >>> HTTP REQUEST GET https://mobilesdk-pa.googleapis.com/v1/projects/703468910137/clients/_:getWebAppConfig  
  
 Fri Feb 23 2018 15:34:30 GMT-0600 (CST)
[debug] [2018-02-23T21:34:32.307Z] <<< HTTP RESPONSE 200
[debug] [2018-02-23T21:34:32.485Z] TypeError: Cannot read property 'public' of undefined
    at Server.<anonymous> (/usr/local/lib/node_modules/firebase-tools/lib/serve/hosting.js:34:18)
    at Object.onceWrapper (events.js:255:19)
    at Server.emit (events.js:160:13)
    at emitListeningNT (net.js:1394:10)
    at process._tickCallback (internal/process/next_tick.js:152:19)
[error] 
[error] Error: An unexpected error has occurred.
